src/main/webapp/profile_creation.jsp

```diff
@@ -0,0 +1,21 @@
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
+<%@taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<t:genericpage>
+    <jsp:body>
+       <div class="content">
+        <form action="/profile_edit" method="POST">
+           <div class="form-group">
+              <label for="username">Username</label>
```

Comment:
Maybe instead of `Username` it should be `Display Name` or `Full Name`, because this is not an unique username and it may confuse the users.